src/System.Net.Sockets/src/System/Net/Sockets/Socket.cs

```diff
@@ -690,27 +690,6 @@ public void Bind(EndPoint localEP)
             if (NetEventSource.IsEnabled) NetEventSource.Exit(this);
         }
 
-        internal void InternalBind(EndPoint localEP)
-        {
-            if (NetEventSource.IsEnabled) NetEventSource.Enter(this, localEP);
-
-            ThrowIfDisposed();
-
-            if (NetEventSource.IsEnabled) NetEventSource.Info(this, $"localEP:{localEP}");
```

Comment:
This PR removes all the tracing for "bind" operations.  Perhaps some of it should be put back in?

Reply:
Added back a trace statement.